buidler.config.js

```diff
@@ -138,17 +138,31 @@ module.exports = {
     }
   },
 
-  solc: {
-    version: "0.6.6",
-    optimizer: require("./solcOptimiserSettings.js")
+  solidity: {
+    compilers: [{
+      version: "0.4.18",
+      settings: {
+        optimizer: require("./solcOptimiserSettings.js")
+      }
+    }, {
+      version: "0.5.11",
+      settings: {
+        optimizer: require("./solcOptimiserSettings.js")
+      }
+    }, {
+      version: "0.6.6",
+      settings: {
+        optimizer: require("./solcOptimiserSettings.js")
+      }
+    }]
   },
 
   paths: {
-    sources: "./contracts/sol6",
+    sources: "./contracts",
     tests: "./test/",
   },
 
   mocha: {
     enableTimeouts: false
```

Comment:
I see some tests time out now
seems this flag not working?

Reply:
Apparently that option is not working anymore. That's weird because, while `enableTimeouts` was removed in mocha@8, buidler depends on version 7.

You can use `timeout: 0` instead.

Reply:
Oh
I see
thanks
can u update here
it fails the CI for this PR